models/user_context.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+import "github.com/google/uuid"
+
+type UserContext struct {
+	Email          string
+	UserID         uuid.UUID `json:"sub"`
+	OrganisationID uuid.UUID `json:"custom:organisationId"`
```

Comment:
Validate that all fields in `UserContext` are correctly populated and consider adding validation logic if necessary to ensure data integrity. [medium]